test-cases/javascript/frameworks/angularjs/index.html

```diff
@@ -0,0 +1,18 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>Security Crawl Maze - AngularJs</title>
+  <base href="/javascript/frameworks/angularjs/">
+  <script src="libs/angularjs/1.8.0/angular.min.js"></script>
```

Comment:
Do you think it would be better to link (https://ajax.googleapis.com/ajax/libs/angularjs/1.8.0/angular.min.js and https://ajax.googleapis.com/ajax/libs/angularjs/1.8.0/angular-route.js) rather than keeping the copies in our repo?

Reply:
Totally agree. Just thought the libs had to be included. I'll change that.